src/common/select/Select.tsx

```diff
@@ -1,47 +1,69 @@
 import React from 'react';
-import classNames from 'classnames';
+import { Dropdown } from 'hds-react';
 
-import styles from './select.module.scss';
+type OptionValue = string | number;
```

Comment:
> type OptionValue = string | number;

Optional: This could be any object if we choose to do deep comparisons. Though restricting it to `string` or `number` covers our current use cases and has a cleaner implementation.

Reply:
The current typing allows specifying enums, so I think this covers our needs well enough. Deep comparison of objects could open some possibilities, but I'm not sure if it has any concrete uses for us right now.  Select options can be mapped from more complex objects